attachments/views.py

```diff
@@ -4,8 +4,11 @@
 from django.template import loader
 from django.utils.encoding import force_text
 from django.views.decorators.csrf import csrf_exempt
+from django.core.mail import send_mail
 
-from attachments.exceptions import VirusFoundException
+from settings import DEFAULT_FROM_EMAIL
```

Comment:
settings are already imported above (`from django.conf import settings`) - you can get rid of this and just use `settings.DEFAULT_FROM_EMAIL` below.